apps/agreement/contracts/example/RegistryApp.sol

```diff
@@ -121,23 +122,29 @@ contract Registry is DisputableAragonApp {
     * @notice Schedule a new entry
     * @dev IForwarder interface conformance
     * @param _data Data requested to be registered
+    * @param _context Link to a human-readable text giving context for the given action
     */
-    function forward(bytes memory _data) public {
-        require(canForward(msg.sender, _data), ERROR_CANNOT_REGISTER);
+    function forward(bytes _data, bytes _context) external payable {
```

Comment:
Isn’t first parameter an EVM script here? That’s confusing to me. Why do we need forwarding then?

Reply:
Yes it is, but I'm playing with it in the Registry app. Although it is a sample app, we can actually not make it forwardable